app/assets/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller.js

```diff
@@ -0,0 +1,287 @@
+          $scope.doApply();
+        }
+      });
+    };
+
+    var getLatestData = function(item) {
+      var params = '&query=get_data&metric_id=' + item.id + '&limit=5&order=DESC';
+
+      $http.get(url + params).success(function (response) {
+        'use strict';
+        if (response.error) {
+          // TODO: do something ?
+        } else {
+          var data = response.data;
+
+          item.lastValues = {};
+          angular.forEach(data, function(d) {
+            item.lastValues[d.timestamp] = formatNumber(d.value);
+          });
+
+          var lastValue = data[0].value;
+          item.last_value = formatNumber(lastValue);
+          item.last_timestamp = data[0].timestamp;
+          if (data.length > 1) {
+            var prevValue = data[1].value;
+            if (angular.isNumber(lastValue) && angular.isNumber(prevValue)) {
+              var change;
+              if (prevValue !== 0 && lastValue !== 0) {
+                change = Math.round(((lastValue - prevValue) / lastValue) * 100.0);
+              } else if (lastValue !== 0) {
+                change = 100;
+              } else {
+                change = 0;
+              }
+              item.percent_change = "(" + change + "%)";
+            }
+          }
+        }
+      });
+    };
+
+    $scope.refresh = function() {
+      $scope.loadingMetrics = true;
+      var _tags = $scope.tags != {} ? '&tags=' + JSON.stringify($scope.tags) : '';
+      $http.get(url + '&query=metric_definitions' + _tags).success(function (response) {
+        'use strict';
+        $scope.loadingMetrics = false;
+        if (response.error) {
+          // TODO: do something ?
+          return;
+        }
+
+        $scope.items = response.metric_definitions.filter(function(item) {
+          return item.tags && item.tags.group_id && item.id && item.minTimestamp;
+        });
+
+        angular.forEach($scope.items, getLatestData);
+
+        $scope.filterConfig.resultsCount = $scope.items.length;
+      });
+    };
+
+    $scope.rangeChange = function() {
+      console.log($scope.timeFilter.time_range);
+    };
+
+    $scope.refresh_graph = function(metric_id, n) {
+      // TODO: replace with a datetimepicker, until then add 24 hours to the date
+      var ends = $scope.timeFilter.date.getTime() + 24 * 60 * 60 * 1000;
+      var diff = $scope.timeFilter.time_range * $scope.timeFilter.range_count * 60 * 60 * 1000; // time_range is in hours
+      var starts = ends - diff;
+      var bucket_duration = diff / 1000 / 100; // bucket duration is in seconds
+      var params = '&query=get_data&metric_id=' + metric_id + '&ends=' + ends + 
+                   '&starts=' + starts+ '&bucket_duration=' + bucket_duration + 's';
+
+      $http.get(url + params).success(function(response) {
+        'use strict';
+        if (response.error) {
+          // TODO: do something ?
+          return;
+        }
+
+        var data  = response.data;
+        var xData = data.map(d => d.start);
```

Comment:
Syntax error! We still need to support es5, please go through this file and update arrow functions to old style functions..